lib/cli.mjs

```diff
@@ -472,7 +473,16 @@ export default async function main(argv) {
 		options.cacheStrategy = cacheStrategy;
 	}
 
-	options.fix = normalizeFixMode(fix);
+	if (isString(fix)) {
+		options.fix = normalizeFixMode(fix);
+
+		// If the fix argument is unknown, consider it as an input path.
+		// TODO: `false` is for backward compatibility. We can remove it in the next major (17.0.0).
+		if (options.fix === undefined && fix !== 'false') {
+			options.fix = DEFAULT_FIX_MODE;
+			cli.input.push(fix);
```

Comment:
question
nit

`input` is "non-flag arguments" which IIUC means that `--fix foo.css --bar=0 qux.css` would now be interpreted as `--fix --bar=0 qux.css foo.css` correct?
Is there any reason why we would want to `unshift` instead?
i.e. start vs end
Would it make a difference?